DOCSP 40401 Node.js counter page (#3313)
mongodb/docs-realm: source/sdk/node/model-data/data-types/field-types.txt

- ``set`` is based on the JavaScript :mdn:`Set <Web/JavaScript/Reference/Global_Objects/Set>` type. ``Realm Set`` is available in the :github:`realm-js@10.5.0 release <realm/realm-js/releases/tag/v10.5.0>`.
- ``mixed`` is a property type that can hold different data types. The ``Mixed`` data type is available in the :github:`realm-js@10.5.0 release <realm/realm-js/releases/tag/v10.5.0>`.
- ``uuid`` is a universally unique identifier from :js-sdk:`Realm.BSON <modules/BSON.html>`. The ``UUID`` data type is available in the :github:`realm-js@10.5.0 release <realm/realm-js/releases/tag/v10.5.0>`.

> - ``counter`` functions as a logical counter when working with synchronized realms. The ``counter`` data type and its associated class is available in the :github:`realm-js@12.10.0 release <realm/realm-js/releases/tag/v12.10.0>`.

Reviewer: ```suggestion
- ``counter`` functions as a logical counter when working with synchronized databases. The ``counter`` data type and its associated class are available in the :github:`realm-js@12.10.0 release <realm/realm-js/releases/tag/v12.10.0>`.
```
